README.md

```diff
@@ -25,6 +25,17 @@ With a Flow pipeline, you:
 -   🌊 **Derive** new collections by transforming from other collections, using
     the full gamut of stateful stream workflow, joins, and aggregations — in real time.
 
+## Get started
+Ready to try out Flow? [Sign up](https://dashboard.estuary.dev/register)
+for free to get started! 🚀
+
+Have questions? We'd love to hear from you:
+
+- Join our [Slack Community](https://go.estuary.dev/slack)
+- Reach out [directly](https://go.estuary.dev/contact-us)
```

Comment:
This "Contact Us" URL doesn't work for me, but omitting the `go` subdomain does work (https://estuary.dev/contact-us).

Reply:
good catch, thank you! updated the URL